### ISSUE TITLE
Add numbers to color palette

### ISSUE THREAD

**Issue by JoshuaAdrianJones** [arcticicestudio/nord#163]
The information for the color palette and image simply says "All colors are numbered from nord0 to nord15 where each palette".

https://raw.githubusercontent.com/arcticicestudio/nord-docs/develop/assets/images/nord/repository-color-palettes.svg?sanitize=true

Could this SVG be updated to show the numbers of each one, this would make adjusting things much easier.

For context the defaults for Emacs Org-Mode TODO in https://github.com/arcticicestudio/nord-emacs/blob/develop/nord-theme.el set some colors and I would like to easily see which of nord`x` I should change some colors to.

**Comment by JoshuaAdrianJones** [comment]
see: https://svgshare.com/s/bdU

**Comment by arcticicestudio** [comment]
Hi @JoshuaAdrianJones :wave:, thanks for your contribution :+1:

Improving the documentations for the individual color IDs has also been raised in #153, but for now I'm not sure if the [color & palette documentation page][1] is not good enough. There are plans to improve these documentations, e.g. by splitting the page into sub-pages for each palette, but right now there are other tasks for the Nord project that must be completed first.

The SVG in the GitHub repository is more like a esthetic element for the visual project representation, even though it also includes HEX numbers already (which can unfortunately not be selected anyway 😕). The main source-of-truth should be the official website with its documentations, that why the README is kept quite minimal when it comes to more detailed information and refers/links to the website instead. "One-page" documentations like a README is often not optimal to document a project in a modular and clear way.

[1]: https://www.nordtheme.com/docs/colors-and-palettes

**Comment by JoshuaAdrianJones** [comment]
closing in favour of #153
